composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "The Magento 2 transformation component",
     "type": "library",
     "require": {
-        "swiftotter/driver": "^3.0.0"
+        "swiftotter/driver": "^3.0.1"
```

Comment:
You might want to make a new release in SwiftOtter/Driver since I merged that PR.

Reply:
Yes, otherwise it won't get update in packagist. Could you do it for me or I'll do it for both repos?

Reply:
Yes, do you not have permission there? I released `v3.1.0` (SemVer)